core/templates/pages/release-coordinator-page/beam-jobs-tab/beam-jobs-tab.component.spec.ts

```diff
@@ -324,13 +325,33 @@ describe('Beam Jobs Tab Component', () => {
 
     fixture.detectChanges();
 
+    // The first time is called by ngOnInit().
     expect(getBeamJobRunsSpy).toHaveBeenCalledTimes(1);
 
     tick(BeamJobsTabComponent.BEAM_JOB_RUNS_REFRESH_INTERVAL_MSECS);
     fixture.detectChanges();
 
+    // The second time is called out by our interval refresh timer.
     expect(getBeamJobRunsSpy).toHaveBeenCalledTimes(2);
 
     component.ngOnDestroy();
   }));
+
+  it('should not refresh beam jobs if all jobs are terminal', fakeAsync(() => {
+    const getBeamJobRunsSpy = spyOn(backendApiService, 'getBeamJobRuns')
+      .and.returnValue(of(terminalBeamJobRuns));
+
+    fixture.detectChanges();
+
+    // The first time is called by ngOnInit().
+    expect(getBeamJobRunsSpy).toHaveBeenCalledTimes(1);
+
+    tick(BeamJobsTabComponent.BEAM_JOB_RUNS_REFRESH_INTERVAL_MSECS);
+    fixture.detectChanges();
+
+    // The second time should not be called out, because all jobs are terminal.
+    expect(getBeamJobRunsSpy).toHaveBeenCalledTimes(1);
+
+    component.ngOnDestroy();
```

Comment:
This seems to be run at the end of each test. Can you place it in an `afterEach` block?

Reply:
Unfortunately not, some tests fail if I do this (the ones closer to the top of the file).